src/helpers/ScanHelper.php

```diff
@@ -75,7 +75,7 @@ public static function scanControllerActionIds(array $controllers)
 				continue;
 			}
 
-			if (! preg_match('/class\s+(([a-z0-9_]+)Controller)[^{]+{/i', $content, $classMatch)) {
+			if (! preg_match('/class\s+(([a-z0-9_]+)Controller)[^{^\*]+{/i', $content, $classMatch)) {
```

Comment:
this one is not correct. You need to clean content from any comments with preg_replace before other checks.
because someone can write:

class Foo /* my class */ {
}

this a valid declaration, but it won't be parsed with this check. to be 100% sure we found correct class declaration we need to remove all comments:

/* ... */
... // some comment "\n"